### DISCUSSION TITLE
Use at my college campus!

### DISCUSSION THREAD

**Discussion by ShantanuXD** [phcode-dev/phoenix#860] [Q&A] [answered]
**Hello**,
i am ShantanuXD, and i wanted to ask if i am allowed to use this program at my college, 
i found this on the web when i was searching for a software which had the capability of showing the code and its results side by side
and i was so amazed by its feature to show the results WITHOUT even installing any other third-party software.
so i hope u agree to my request and let me and my classmates study coding.

**Comment by abose** [comment, marked as answer]
Hi @ShantanuXD 

Phoenix is free software licensed under GNU AGPl. Feel free to use it anywhere and for any purpose. 

Do raise issues or discuss here if you would like any features editor in editor too

**Reply by ShantanuXD** [reply]
Thank your Very much!!!
